tel2puml/tel2puml_types.py

```diff
@@ -144,6 +157,7 @@ class PVPumlOptions(TypedDict):
     file_list: list[str]
     job_name: str
     group_by_job_id: bool
+    mapping_config: MappingConfig | None
```

Comment:
change this to just MappingConfig and also NotRequired

Reply:
what do you mean change it to just MappingConfig here?

Reply:
The typing i.e. without None then surround with NotRequired